src/routes/committee/rehearsals/[rehearsalId]/availability/+page.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+	import {
+		Column,
+		Grid,
+		Row,
+		StructuredListBody,
+		StructuredListCell,
+		StructuredListHead,
+		StructuredListRow
+	} from 'carbon-components-svelte'
+	import type { PageData } from './$types'
+	import ScrollableList from '$lib/components/scrollableList.svelte'
+	import type { availability } from '$lib/types/domain/availability'
+
+	export let data: PageData
+
+	const usersWithUndefinedAvailabilities = data.userAvailabilities.filter((x) => !x.availability)
+	const usersWithDefinedAvailabilities = data.userAvailabilities.filter((x) => !!x.availability)
+	const unavailableUsers = usersWithDefinedAvailabilities.filter((x) => !x.availability!.available).map(
+		(user) => {
+			return {
+				name: user.name,
+				reason: user.availability!.reason ? user.availability!.reason : 'No reason given'
+			}
+		}
```

Comment:
```suggestion
		(user) => ({
			name: user.name,
			reason: user.availability!.reason ? user.availability!.reason : 'No reason given'
		})
```
		also change in const below this one